models/localized-string/src/types.ts

```diff
@@ -0,0 +1,17 @@
+import type { TBuilder } from '@commercetools-test-data/core';
+
+export type TLocalizedStringBuilder = TBuilder<TLocalizedString>;
+
+export type TCreateLocalizedStringBuilder = (args?: {
+  defaults?: Partial<TLocalizedString>;
+}) => TLocalizedStringBuilder;
+
+export type TLocalizedString = {
+  [locale: string]: string | undefined;
```

Comment:
Because of the "empty" preset, where we set the value to `undefined`, we also need to define it here as a type.

Q: is this something we have to support and is it how it's meant to be?

Reply:
Defines on the definition of empty I assume. If empty is explicitly set to `undefined` we, I think need it.